### PR TITLE
Fix GGOV1

### PR DESCRIPTION
**Pull request by marcelofcastro**
This PR should fix the issues with GGOV1 and thus solve issues #119 and #226.

### CODE REVIEW

File: OpenIPSL/Electrical/Controls/PSSE/TG/BaseClasses/GGOV1/AccelerationLimiter.mo [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+<td><p>Last update</p></td>
+<td><p>November 2020</p></td>
+</tr>
+<tr>
+<td><p>Author</p></td>
+<td><p>ALSETLab, Rensselaer Polytechnic Institute</p></td>
+</tr>
+<tr>
+<td><p>Contact</p></td>
+<td><p><a href=\"mailto:vanfrl@rpi.edu\">vanfrl@rpi.edu</a></p></td>
```
Comment (dietmarw) [inline OpenIPSL/Electrical/Controls/PSSE/TG/BaseClasses/GGOV1/AccelerationLimiter.mo:85]: Please point to `OpenIPSL.UsersGuide.Contact` in all places.

Comment (marcelofcastro) [reply OpenIPSL/Electrical/Controls/PSSE/TG/BaseClasses/GGOV1/AccelerationLimiter.mo:85]: Is there a model you've done this? so I can use it as a template...

Comment (dietmarw) [reply OpenIPSL/Electrical/Controls/PSSE/TG/BaseClasses/GGOV1/AccelerationLimiter.mo:85]: Yes, look at `OpenIPSL.Electrical.Branches.PSAT.PhaseShiftingTransformer`. Also remember that the table should go in the revison section and not the info section.

**Review by dietmarw — CHANGES_REQUESTED** [review]
Here some things that should be changed:
 1. clean up the example. Have a look at how the other examples right next to it look like wrt size and positioning and routing. Also **never** stretch the icons. It just looks ugly. 
 2. Clean up positioning of all components in all the new models. 
  - Try to align on the grid. 
  - Make sure that connectors are always positioned along the edge. Basically make it look tidy and less chaotic. 
  - Try **hard** to only use the default 200x200 real estate also on the diagram layer. 
Just have a look at `Modelica.UsersGuide.Conventions.Icons` which is not a must but shows some basic (and thought through) rules when it comes to where to place what.

**Comment by marcelofcastro** [comment]
@dietmarw I think I did what you asked.

**Review by dietmarw — APPROVED** [review]
This is now fine to merge from my side. One thing though, the examples simulate (in Dymola) still with quite some numerical solver errors which seem to point to unfortunate paramterisation. Not sure if there is something that we can do about that.

**Comment by marcelofcastro** [comment]
I noticed that. I think it might be something related to our Types stuff. We didn't have any issue before so I guess it is something we might take a look when fixing it. I will merge this PR.

**Pull request merged**